ci/sign-all-tasks.yml

```diff
@@ -0,0 +1,35 @@
+parameters:
+  layoutRoot: ''
+
+steps:
+# Guard against missing parameters
+- ${{ if eq(parameters.layoutRoot, '') }}:
+  - |
+      Badly configured build; an expected parameter is missing. //
+      layoutRoot: ${{ parameters.layoutRoot }}
```

Comment:
This idiom isn't needed. Define the parameter thusly:
```yaml
parameters:
- name: layoutRoot
  type: string
```
Now it will be a compilation error if a layoutRoot isn't provided by the caller.

Reply:
Thanks Mark!